docs/UserGuide.md

```diff
@@ -113,7 +113,7 @@ Format: `list`
 
 Edits an existing guest in the guest book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [rm/ROOM] [dr/DATE_RANGE] [ng/NUMBER_OF_GUESTS] [rc/IS_ROOM_CLEAN] ​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [rm/ROOM] [dr/DATE_RANGE] [ng/NUMBER_OF_GUESTS] ​`
```

Comment:
Could we remove the space at the end of the code block?

Reply:
Fixed!